crates/sui-framework-tests/tests/move_tests.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(not(msim))]
```

Comment:
Do we need the msim cfgs?

Reply:
I can try without, I borrowed them from the PTB tests.

Reply:
It works fine without! It does get re-run during simtest, which is not strictly necessary, but that seems okay.